test/e2e/configuration_policy_test.go

```diff
@@ -20,24 +20,30 @@ import (
 func configPolicyTestCleanUp(roleName, rolePolicyName, rolePolicyYAML string) {
 	By("Deleting the role, policy, and events on managed cluster")
 	common.DoCleanupPolicy(rolePolicyYAML, common.GvrConfigurationPolicy)
-	_, err := common.OcManaged(
-		"delete", "events", "-n", "managed",
+	_, err := common.OcHosting(
+		"delete", "events", "-n", common.ClusterNamespace,
 		"--field-selector=involvedObject.name="+common.UserNamespace+"."+rolePolicyName,
 		"--ignore-not-found",
 	)
 	ExpectWithOffset(1, err).To(BeNil())
-	_, err = common.OcHub(
-		"delete", "events", "-n", "managed",
-		"--field-selector=involvedObject.name="+common.UserNamespace+"."+rolePolicyName,
+	_, err = common.OcHosting(
+		"delete", "events", "-n", common.ClusterNamespace,
+		"--field-selector=involvedObject.name="+rolePolicyName,
 		"--ignore-not-found",
 	)
 	ExpectWithOffset(1, err).To(BeNil())
-	_, err = common.OcManaged(
-		"delete", "events", "-n", "managed",
+	_, err = common.OcHub(
+		"delete", "events", "-n", common.UserNamespace,
```

Comment:
Did you mean this?
```suggestion
	_, err = common.OcHosting(
		"delete", "events", "-n", common.ClusterNamespace,
```

Reply:
I was trying to remove this event `policy-test       role-policy-musthave               inform               NonCompliant       51s`

Reply:
There shouldn't be any events published to `UserNamespace` though. It should only be in the managed cluster namespace.

Reply:
let me test without that let you know